lib/customisation.nix

```diff
@@ -365,6 +378,35 @@ rec {
     else
       mapAttrs mkAttrOverridable pkgs;
 
+  makeAlias =
+    let
+      currentRelease = lib.versions.majorMinor lib.version;
+    in
+    self: name:
+    {
+      type,
+      package ? null,
+      reason ? null,
+      release ? null,
```

Comment:
I have a mild preference for `since` over `release` here – it seems less ambiguous that it’s when the alias was added, rather than when it’s due for complete removal.